src/main/java/com/attacktimer/AttackTimerMetronomeTileOverlay.java

```diff
@@ -1,9 +1,10 @@
-package com.attacktimer;
```

Comment:
I don't think we want to change the package name here.

Reply:
The package name automatically adjusted due to it being compiled in my intelij.

I imagine if you swap it back to the original line it would be fine.

Reply:
Can you swap the name back in your PR?

Reply:
I will give it a shot, was busy making some other things Ill try to get around to it tonight :)